presto-main/src/main/java/com/facebook/presto/operator/ChannelSet.java

```diff
@@ -100,16 +108,34 @@ public int size()
             return hash.getGroupCount();
         }
 
-        public void addPage(Page page)
+        /**
+         * This function adds a page to a work object for later processing.
+         * @param page A data page whose data needs to be added to the hash table.
+         * @return The work needs to be done on this page.
```

Comment:
`The work that needs to be done` or `Pending work`?

Reply:
I think they're the same, but I removed the javadoc comment anyways